### PR TITLE
Add definitions

### PR DESCRIPTION
**Pull request by joschicif**

### CODE REVIEW

File: templates/codeTable.mediawiki [hunk 1]
```diff
@@ -1,2 +1,73 @@
...
 
+==Blockchain/Distributed Ledger Protocol==
+
+* BigChainDB
+* Bitcoin
+* Chain
+* Corda
+* EOS
+* Ethereum
+* Hyperledger
```
Comment (Chainomics) [inline templates/codeTable.mediawiki]: Hyperledger Fabric

File: templates/codeTable.mediawiki [hunk 2]
```diff
@@ -1,2 +1,73 @@
...
+Note: If you are interested in contributing to this project, please make sure that you use the following framework and definitions.
 
+==Blockchain/Distributed Ledger Protocol==
+
+* BigChainDB
+* Bitcoin
+* Chain
+* Corda
+* EOS
+* Ethereum
```
Comment (Chainomics) [inline templates/codeTable.mediawiki]: * Ethereum Main Chain
* Ethereum Private Chain

File: templates/codeTable.mediawiki [hunk 3]
```diff
@@ -1,2 +1,73 @@
-=Code Table=
+=Definitions=
+Note: If you are interested in contributing to this project, please make sure that you use the following framework and definitions.
 
+==Blockchain/Distributed Ledger Protocol==
+
+* BigChainDB
+* Bitcoin
+* Chain
```
Comment (Chainomics) [inline templates/codeTable.mediawiki]: ?

File: templates/codeTable.mediawiki [hunk 4]
```diff
@@ -1,2 +1,73 @@
...
+* Test (Proof of concept)
+* Live
+
+==Token==
+
+* Asset token
+* Currency token
+* Equity token
+* Reputation and reward token
+* Utility token
```
Comment (Chainomics) [inline templates/codeTable.mediawiki]: Maybe too complicated and hard to distinguish. I'd rather go with a simple definition like:
* Utility/payment token
* Security token
where utility/payment tokens do not include any promises for off-chain assets, payments or services.

Comment (Chainomics) [reply templates/codeTable.mediawiki]: maybe call it security-like token in order to make sure there are no legal hiccups...

File: templates/codeTable.mediawiki (hunk 3 above)
Comment (joschicif) [reply templates/codeTable.mediawiki]: [This](https://chain.com/) is the website of chain. Do you want me to take it out again?

**Comment by joschicif** [comment]
Thanks for the review! I will create a new PR as soon as possible (tonight or tomorrow).

File: templates/codeTable.mediawiki (hunk 3 above)
Comment (Chainomics) [reply templates/codeTable.mediawiki]: No that's fine. No worries. Thanks for the link.

**Comment by Chainomics** [comment]
@joschicif No need for a new pr. Just commit to this branch and the new commit will get added to the existing pr.

**Pull request merged**